### PR TITLE
move simulator entrypoint

### PR DESCRIPTION
**Pull request by YamasouA**
#### What this PR does / why we need it:
There are Issue that separate API endpoint and Scheduler.
First I try to move simulator entry points. (next step try to add scheduler entry points to kube-scheduler-simulator/simulator/cmd/scheduler)

#### Which issue(s) this PR fixes:

Part of #318 

#### Special notes for your reviewer:

/label tide/merge-method-squash

### CODE REVIEW

**Review by sanposhiho — COMMENTED** [review]
We have to modify [Dockerfile](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/simulator/Dockerfile)

File: simulator/cmd/simulator/config/config.go
Comment (sanposhiho) [inline simulator/cmd/simulator/config/config.go:1]: Any reason you moved simulator/config/ under /cmd?

Comment (YamasouA) [reply simulator/cmd/simulator/config/config.go:1]: Thank you for the review!
On second thought, there was no need to move it.

**Comment by YamasouA** [comment]
I fix Docker file and pass `docker build`.

**Review by sanposhiho — COMMENTED** [review]
/ok-to-test

**Review by sanposhiho — APPROVED** [review]
Thanks.

/lgtm
/approve

**Pull request merged**